src/components/AppLayout/ListFilters/ListFilters.tsx

```diff
@@ -33,9 +33,6 @@ export const ListFilters = ({
       gap={7}
       paddingBottom={5}
       paddingX={9}
-      borderColor="neutralPlain"
-      borderBottomStyle="solid"
-      borderBottomWidth={1}
```

Comment:
question: don't we need this border anymore?
<img width="1490" alt="Screenshot 2023-04-21 at 11 30 40" src="https://user-images.githubusercontent.com/16362049/233603264-7e50ee14-2c53-4949-813c-2c09f4dbf96c.png">

Reply:
@poulch this border is redundant